mesonbuild/environment.py

```diff
@@ -1739,7 +1738,10 @@ def detect_rust_compiler(self, for_machine):
                 elif 'link' in override[0]:
                     linker = self._guess_win_linker(
                         override, RustCompiler, for_machine, use_linker_prefix=False)
+                    # rustc takes linker arguments undercorator, and does
```

Comment:
"undecorated"?

Reply:
yeah, that should be "undecorated", but maybe "Without a prefix" would be better, since we generally refer to the (for example) `-Wl,` that gcc uses to forward linker args as a prefix.